docs/cmdline_preface.md

```diff
@@ -0,0 +1,16 @@
+> [!NOTE]
+> This file is automatically generated by `make`.
+> Please modify src/cmd/*.rs files and rebuild cros to update.
```

Comment:
Did you mean `rebuild cro3`?

Besides, I'm wondering who is the intended reader of this NOTE. Maybe something like "If you have modified src/cmd/*.rs files, rebuild cro3 to update this page"?

Reply:
Ah, good catch... Fixed! Also, updated the text to make it better. Thank you!